scripts/qemu/ia32-generic.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+#
+# Shell script for running Phoenix-RTOS on QEMU (ia32-generic)
+#
+# Copyright 2021 Phoenix Systems
+# Author: Lukasz Kosinski
+#
+
+DIR="$(dirname "$BASH_SOURCE")"                          # Script directory
+DRIVES="-hda $DIR/../../_boot/phoenix-ia32-generic.disk" # Default drives
```

Comment:
Unfortunately this is not the correct way for safely escaping multiple params - you need to use bash array (this applies also to every other param  with spaces :/).

The easiest way to trigger the problem is to exec with "dir with spaces" in path:
```bash
wm@hatifnat:~/src$ "./dir with spaces/phoenix-rtos-project/scripts/qemu/ia32-generic.sh"
qemu-system-i386 -hda ./dir with spaces/phoenix-rtos-project/scripts/qemu/../../_boot/phoenix-ia32-generic.disk -vga std  -serial stdio
qemu-system-i386: -hda ./dir: Could not open './dir': No such file or directory
```

This is reported by `shellcheck` as [SC2086](https://www.shellcheck.net/wiki/SC2086)
qemu-system-i386: -hda ./dir: Could not open './dir': No such file or directory

Reply:
Fixed in [next PR](https://github.com/phoenix-rtos/phoenix-rtos-project/pull/58).